how-tos/merge-day.md

```diff
@@ -149,16 +148,22 @@ python mozharness-central/scripts/merge_day/gecko_migration.py \
 ssh buildbot-master01.bb.releng.use1.mozilla.com
 sudo su - cltbld
 cd /builds/l10n-bumper
-mozharness/scripts/l10n_bumper.py -c configs/l10n_bumper/mozilla-beta.py --ignore-closed-tree
+python2.7 mozharness/scripts/l10n_bumper.py -c mozharness/configs/l10n_bumper/mozilla-beta.py --ignore-closed-tree
 ```
 1. Verify changesets are visible on [hg pushlog](https://hg.mozilla.org/releases/mozilla-beta/pushloghtml) and [Treeherder]( https://treeherder.mozilla.org/#/jobs?repo=mozilla-beta). It may take a couple of minutes to appear.
-1. Tell sheriffs this migration is done. They will trigger new nightly builds? ([jlorenzo] unsure of that part)
+1. Ping l10n folks in #releng (usually `Pike`) and send them a heads-up that RelEng side of migration is completed
+1. Trigger nightlies for central. This can be done by:
+    1. trigger [nightly-desktop/mozilla-central](https://tools.taskcluster.net/hooks/#project-releng/nightly-desktop%252fmozilla-central) hook
+    1. trigger [nightly-fennec/mozilla-central](https://tools.taskcluster.net/hooks/#project-releng/nightly-fennec%252fmozilla-central) hook
+    1. go to [BuildAPI](https://secure.pub.build.mozilla.org/buildapi/self-serve/mozilla-central) and look for `Create new nightly builds on mozilla-central revision`. Fill in the latest nightly revision and trigger the nightlies!
+1. Tell sheriffs this migration is done either by popping a short message in #sheriffs or ping the `<handle>|sheriffduty` person from #releng.
 
 ### Update wikis
 
-1. 
```

Comment:
I think it'll stay the same. Wrong, but the same. It's quite weird, I've tested with pandoc and I've got completely different results on my local machine. Bleah, I'll merge like this and follow-up afterwards if we have to 👍 https://github.com/mozilla/releasewarrior/blob/master/how-tos/merge-day.md#update-wikis and https://github.com/MihaiTabara/releasewarrior/blob/improvements/how-tos/merge-day.md look somehow